thirdparty/runtime_environments.go

```diff
@@ -365,13 +366,14 @@ func stringToTime(timeInitial string) (time.Time, error) {
 	return time.Unix(timestamp, 0), nil
 }
 
-// Image stores information about an image including its name, version ID, kernel, AMI, and its last deployed time.
+// Image stores information about an image including its AMI, imageID, kernel, last deployed time, name, and versionID.
 type Image struct {
-	Name         string
-	VersionID    string
+	AMI          string
+	ImageID      string
```

Comment:
`Image.ID` is more idiomatic than `Image.ImageID`
```suggestion
	ID      string
```

Reply:
done!